test/blackbox-tests/test-cases/coq/compose-boot-plugins.t/dune-workspace

```diff
@@ -0,0 +1 @@
+(lang dune 3.4)
```

Comment:
Don't we need `(using coq 0.5)` here as well? Seems like a bug.

Reply:
I don't think so, but not expert on workspaces.